airlock/business_logic.py

```diff
@@ -868,9 +866,7 @@ def register_file_upload(
             user=user,
             path=relpath,
         )
-        self._dal.register_file_upload(
-            release_request.id, relpath, user.username, audit
-        )
+        self._dal.register_file_upload(release_request.id, relpath, audit)
```

Comment:
Because this was one of the few methods where the number of arguments had changed I spent ages failing to understand how this code could possibly work and then I realised I was mixing up the DAL and BLL methods!

Reply:
Yeah, I think I just caught an unneeded parameter in a DAL call that could be removed, because we already construct the AuditEvent in the BLL, and that's all this method needs.

In hindsight, I probably could have pulled out the removal of the username parameter from this DAL method in a separate commit